Documentation/users/ldap-integration.md

```diff
@@ -0,0 +1,144 @@
+  * **Username Attribute**: The end user's username for login. For use with Kubernetes, the Username Attribute must be set to the user's email address.
+  * **User ID Attribute**: String representation of the user's [unique identifier][uid-ldap].
+  * **Email Attribute**: Required. Attribute to map to email. For use with Kubernetes, this value must be set to something other than the user's email address.
+  * **Name Attribute**: The user's display name.
+
+5. Click *Continue* to enter Group information:
+
+  * **Base DN**: The root LDAP directory to begin the group search. Translates to the query: `(&(objectClass=group)(member=<user uid>))`.
+  * **Filter**: (Optional) Filter to apply when searching the directory. For example, when a group search executes, object results could be limited to those with an `objectClass` of [groupOfNames][groupOfNames-ldap].
+  * **User Attribute**: The user field that a group uses to identify that a user is part of a group. For example, groups that specify each member as `member: cn=john,dc=example,dc=org` in the LDAP directory, are using the [Distinguished Name (DN)][dn-ldap] attribute.
+  * **Member Attribute**: The [member][member-ldap] field associating a user with a group, using the User Attribute mentioned above.
+  * **Name Attribute**: The group's display name.
+
+6. Click *Continue* to enter a test *Username* and *Password*, then click *Test Configuration* to verify that users and groups are correctly configured.
+
+7. When confirmed, click *Continue*, and follow the instructions to backup the existing, then apply the new configuration to the cluster.
+
+## Applying LDAP Config changes
+
+First, click **Download Existing Config** to download a backup of the existing configuration.
+
+Then, click **Download New Config** to download the new configuration YAML file.
+
+Run `kubectl apply` to apply the new configuration:
+
+```
+kubectl apply -f path/to/config/file/new-tectonic-config.yaml
+```
+
+Finally, trigger a rolling update of the Identity pods, which will read the new configuration:
+
+```
+kubectl patch deployment tectonic-identity \
+    --patch "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"date\":\"`date +'%s'`\"}}}}}" \
+    --namespace tectonic-system
+```
+
+If successful the following message is displayed:
+
+```
+"tectonic-identity" patched
+```
+
+## Using kubectl as an LDAP user
+
+To use `kubectl` as an LDAP user:
+
+1. Log in to Tectonic Console as the desired user.
+2. From *My Account*, click *KUBECTL: Download Configuration*.
+3. Set the `KUBECONFIG` environment variable to the kubectl configuration file. For example:
+
+`export KUBECONFIG=~/Download/kubectl-config`
+
+Until otherwise modified, use your static account for further administrative setup.
+
+## Example schema
+
+The following examples demonstrate mapping LDAP schema to Tectonic Identity configurations.
+
+### User LDAP schema and Tectonic mapping
```

Comment:
nit: this section and the one below are both examples of the same functionality: determining what group a user's in. Maybe combine the two sections?

Reply:
i don't think i understand LDAP schema well enough to make a clean attempt at this. Do the two LDAP snippets both show a member definition at the top:
```
dn: cn=john,dc=people,dc=example,dc=org
objectClass: person
cn: jane
email: jane.doe@example.com
```
followed by a group definition at the bottom:
```
dn: cn=developers,dc=groups,dc=example,dc=org
objectClass: group
cn: developers
member: jane
```

that is, are these simply two different ways to define group affiliations?